www/components/shelf/getting-started-list.json

```diff
@@ -18,7 +18,7 @@
     }, {
       "index": 2,
       "name": "Project Structures",
-      "id": "project-structures"
+      "id": "project-structure"
```

Comment:
was it intentional to change the `id`?  Doesn't the `name` need to match to the `id` for in page scrolling?

Reply:
the id has to match the header yes. That's why I changed it. It doesn't work currently.

Reply:
oh, the heading in the document.  got it.  So what is the `name` for?  If we're making everything else singular, shouldn't we change `name` too?

Reply:
Name is for the text of the shelf link. We can also have be structure(singular) if you'd like but it has no affect on the link.

Reply:
I don't think it makes sense to have them be different then?  The left nav is one way, but the heading is another?  I would see that as a typo / grammar error.  

Can we make them all the same?

Reply:
Singular or plural? Which is the more accurate heading name?  If plural, we can revert this change and fix the getting-started page for project-structure(s).  If singular, I need to add singular to the name of this shelf link.

Reply:
i think it was intended to be _Project Structure_ (singular)